.changeset/funny-garlics-double.md

```diff
@@ -0,0 +1,8 @@
+---
+'eslint-config-seek': patch
+---
+
+Replace `eslint-plugin-cypress` with the [`@finsit/eslint-plugin-cypress`] fork that supports ESLint v8.
+Consumers that were overriding `cypress` rules will need instead override `@finsit/cypress` rules.
```

Comment:
Am I correct that this could be considered a breaking change, but that we aren't due to the low impact and the type of library this is?

Reply:
Correct. I can add a note to the changeset about this.